cityofaustin/atd-moped
# Fixes duplicate contract items in projects list

## Associated issues
Fixes https://github.com/cityofaustin/atd-data-tech/issues/10485

This was not a regression—just a lingering bug from v1.9 that we didn't catch.

## Testing
**URL to test:** 
Local, please.

**Steps to test:**
1. Add three contracts to a project via the funding tab. Include a contractor name and contract number.
2. Delete one of the contracts.
3. Verify the two contractors and contract #'s appear in the project list view (you may have to enable the columns).

---
#### Ship list
